README.md

```diff
@@ -156,7 +156,17 @@ https://192.168.37.1:8443/console/
 
 #### Step 3.1, Add a mobile app
 
-In your project, click on the mobile tab located in the left hand navigation. Next click ```create app``` fill in some details and pick cordova as the type. Then click create. (note) you will need to accept the cert of the mobile server. You can do this by hitting https://localhost:3001 in your browser.
+In your project (the `localmcp` project), click on the mobile tab located in the left hand navigation. 
+
+##### Provision the Mobile Control Panel
+
+For the first time it is needed to provision the Mobile Control Panel, for that click on the `Provision Mobile Control Panel` button and pick the `mcp` item from the Service Catalog. Select your project and move on. Once finished click the `View Project` button which brings you back to your project's overview. 
```

Comment:
This isn't needed for local development. For local development, you're running the mcp server locally at https://localhost:3001

Reply:
I do get the screen that asks me to provision it :thinking:

Reply:
That screen shows if:
* the UI can't find a Route called 'mcp-standalone' (i.e. MCP server wasn't provisioned to the namespace)
* the UI is configured to use a locally running MCP server & there is connectivity trouble talking to it (e.g. not running or cert not accepted in browser)

For local development, it would be the latter.
Local development doesn't require an MCP server to be provisioned in the namespace (despite the counter-intuitive message in the UI). If it isn't working for local development, there is some other problem.

Reply:
> the UI is configured to use a locally running MCP server & there is connectivity trouble talking to it (e.g. not running or cert not accepted in browser)

ha, now I get it, and that's what I running into (or ran) 

let me reword that on the train ;-)